package.json

```diff
@@ -0,0 +1,85 @@
+{
+  "name": "reviews",
+  "version": "0.0.0",
+  "description": "Center for Open Science Reviews Service",
+  "license": "MIT",
+  "author": "",
+  "directories": {
+    "doc": "doc",
+    "test": "tests"
+  },
+  "repository": "",
+  "scripts": {
+    "build": "ember build",
+    "start": "ember server",
+    "test": "ember test"
+  },
+  "devDependencies": {
+    "autoprefixer": "6.3.7",
```

Comment:
Citing Barrett again, can you remove all the deps and add them back in as required?